### PR TITLE
LEAF-1406: Fixed dialog width

### PR DESCRIPTION
**Pull request by abiola-adewuyi**
Updated width of dialog box used in the history pop up dialogs.

### CODE REVIEW

File: LEAF_Nexus/templates/site_elements/generic_dialog.tpl [hunk 1]
```diff
@@ -3,6 +3,6 @@
         <div id="genericDialogbutton_cancelchange" style="display: none"></div>
         <div id="genericDialogbutton_save" style="display: none"></div>
         <div id="genericDialogloadIndicator" style="visibility: hidden; z-index: 9000; position: absolute; text-align: center; font-size: 24px; font-weight: bold; background-color: #f2f5f7; padding: 16px; height: 400px; width: 526px"><img src="images/largespinner.gif" alt="loading..." /></div>
-        <div id="genericDialogxhr" style="width: 540px; height: 420px; padding: 8px; overflow: auto; font-size: 12px"></div>
+        <div id="genericDialogxhr" style="min-width: 540px; min-height: 420px; width: 100%; height: 100%; padding: 8px; overflow: auto; font-size: 12px"></div>
```
Comment (dastan-rahimi) [inline LEAF_Nexus/templates/site_elements/generic_dialog.tpl:6]: Having height at 100% is causing the dialog to expand to the whole screen and cuts off some content in form editor section.

**Comment by dastan-rahimi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/VALEAF/LEAF/pullRequest?prid=5349352&bid=17516462)

**Comment by abiola-adewuyi** [comment]
The Codacy check is failing because the CSS "box-sizing" is not supported in IE 6 & 7. I don't believe we support those browser versions so I'll suggest this check is ignored and the pull request approved.

**Comment by dastan-rahimi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/VALEAF/LEAF/pullRequest?prid=5349352&bid=17516462)

**Pull request merged**